types/model.ts

```diff
@@ -0,0 +1,13 @@
+// These are temporary for now - will be updated once we nail down the backend models
```

Comment:
we can probably just rename this to Song.ts and move it to the types/16bars folder like the other entity models here 
![image](https://user-images.githubusercontent.com/39350987/107112129-36f37100-6823-11eb-8a47-5a8546a20753.png)

Reply:
Yeah that's fair - should be easy enough to move around later, can leave it like this for now.